docs/topics/Execute.adoc

```diff
@@ -204,11 +204,11 @@ WARNING: If you omit the `--packages` argument, every package in the application
 [[source_argument]]
 --source SOURCE_1 SOURCE_2:: A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. This argument, in conjunction with the `--target` argument, helps to determine which rulesets are used. Use the `--listSourceTechnologies` argument to list all available sources.
 +
-The `--source` argument now supports the link:http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html[Maven version range syntax], which allows you to run the rules appropriate for your specific technology version. For example, `--source eap:5`.
+The `--source` argument now provides version support, which follows the link:http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html[Maven version range syntax]. This instructs Windup to only run the rulesets matching the specified versions. For example, `--source eap:5`.
 +
 [WARNING]
 ====
-When migrating from an earlier JBoss EAP version to a newer one, be sure to specify the version for the source and target, for example, `eap:6`. Specifying only `eap` may run both JBoss EAP 6 and JBoss EAP 7 rulesets, which can cause duplicate hits.
+When migrating from an earlier JBoss EAP version to a newer one, be sure to specify the version for the source and target, for example, `eap:6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
```

Comment:
Good catch @jsight . I fixed the note in the --target section to be more generic to apply to competitive as well, but missed updating this one. I'll make the update, thanks!